include/dsn/dist/partition_resolver.h

```diff
@@ -68,7 +68,12 @@ namespace dsn
             typedef partition_resolver* (*factory)(rpc_address&, const char*);
 
         public:
-            partition_resolver(rpc_address meta_server, const char* app_path);
+            partition_resolver(rpc_address meta_server, const char* app_path)
```

Comment:
Moving inline implementation at the bottom of the .h file is actually a better coding style from my perspective. Seems it is time for rDSN to come up a consistent coding convention.